custom-namespaces.yaml

```diff
@@ -711,6 +715,11 @@ firefox_desktop:
       tables:
         - table: mozdata.default_browser_agent.default_browser
   explores:
+    urlbar_events:
+      type: events_explore ???
+      views:
+        base_view: urlbar_events ???
+        extended_view: ???
```

Comment:
@scholtzan I'm trying to add a new derived table where 1 row represents 1 Glean event. Could you help me with the values here? I'm not sure what the valid values are and what they mean.